src/reporting/disputeBondToken.se

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+
+#### must be first data members to match delegator.se
+data controller
+data controllerLookupName
+####
+
+inset('../macros/externs.sem')
+inset('../macros/safeMath.sem')
+inset('../macros/assertNoValue.sem')
+inset('../macros/require.sem')
+
+data initialized
+data market
+data bondHolder
+data payoutDistributionHash #Disputed outcome
+data bondRemainingToBePaidOut
+data tokens[2**160]
+data spenders[2**160][2**160]
+data supply
+
+def any():
+    assertNoValue()
+
+def initialize(market: address, bondHolder: address, bondAmount: int256, payoutDistributionHash: int256):
+    require(not self.initialized)
+    self.initialized = 1
+    self.market = market
+    self.bondHolder = bondHolder
+    self.payoutDistributionHash = payoutDistributionHash
+    self.bondRemainingToBePaidOut = bondAmount * 2
+    self.market.getReputationToken().transferFrom(msg.sender, self, bondAmount)
```

Comment:
2**254 is just about the largest number Ethereum can handle, and since we explicitly trust the market, its simplest to set this as high as possible.  Each time market transfers funds on behalf of disputeBond, the amount being transferred is deducted from that total and we _really_ don't want it to run out.